manifests/scenarios/gmp-otel/1_collector.yaml

```diff
@@ -169,55 +171,90 @@ data:
         detectors: [gcp]
         timeout: 10s
 
-      transform/collision:
-        metric_statements:
-        - context: datapoint
-          statements:
-          - set(attributes["exported_location"], attributes["location"])
-          - delete_key(attributes, "location")
-          - set(attributes["exported_cluster"], attributes["cluster"])
-          - delete_key(attributes, "cluster")
-          - set(attributes["exported_namespace"], attributes["namespace"])
-          - delete_key(attributes, "namespace")
-          - set(attributes["exported_job"], attributes["job"])
-          - delete_key(attributes, "job")
-          - set(attributes["exported_instance"], attributes["instance"])
-          - delete_key(attributes, "instance")
-          - set(attributes["exported_project_id"], attributes["project_id"])
-          - delete_key(attributes, "project_id")
+      #transform/collision:
+      #        metric_statements:
+      #        - context: datapoint
+      #          statements:
+      #          - set(attributes["exported_location"], attributes["location"])
+      #          - delete_key(attributes, "location")
+      #          - set(attributes["exported_cluster"], attributes["cluster"])
+      #          - delete_key(attributes, "cluster")
+      #          - set(attributes["exported_namespace"], attributes["namespace"])
+      #          - delete_key(attributes, "namespace")
+      #          - set(attributes["exported_job"], attributes["job"])
+      #          - delete_key(attributes, "job")
+      #          - set(attributes["exported_instance"], attributes["instance"])
+      #          - delete_key(attributes, "instance")
+      #          - set(attributes["exported_project_id"], attributes["project_id"])
+      #          - delete_key(attributes, "project_id")
 
     receivers:
       prometheus/bench:
```

Comment:
[nit] Lets add a comment here about why this scrape config is so complicated - it tries to replicate exactly what the operator creates